### ISSUE TITLE
Missing package product 'MessageKit'

### ISSUE THREAD

**Issue by ParitaPatel18** [MessageKit/MessageKit#1614]
**Describe the bug**

**To Reproduce**
Steps/code to reproduce the behavior:

**Expected behavior**

**Screenshots**

**Environment**
- What version of MessageKit are you using?
- What version of iOS are you running on?
- What version of Swift are you running on?
- What device(s) are you testing on? Are these simulators?
- Is the issue you're experiencing reproducible in the example app?

**Additional context**

**Comment by mironal** [comment]
I got the same error. 😢

**Comment by Kaspik** [comment]
This Issue is missing a lot of details. Could you please fill up the form again?

**Comment by tomblah** [comment]
I got the same error message as OP when attempting to run the example app out of the box.

I solved it by banging my head against the wall:

(1) Removed MessageKit from "Frameworks, Libraries and Embedded Content"
(2) Tapped "+" button
(3) Chose "Add Other"
(4) Chose "Add Package Dependency"
(5) Pasted in https://github.com/MessageKit/MessageKit
(6) Followed the rest of the wizard.

And voila! Plesantly surprised that it actually worked :-)

NB: I usually use CocoaPods as opposed to Swift Package Manager, so I'm not really sure what I was doing.
